docs/Installation/Installation.md

```diff
@@ -25,7 +25,7 @@ environment differently, read on!
 * [Charm++](http://charm.cs.illinois.edu/) 6.10.2
 * [Git](https://git-scm.com/)
 * BLAS (e.g. [OpenBLAS](http://www.openblas.net))
-* [Blaze](https://bitbucket.org/blaze-lib/blaze/overview) v3.5
+* [Blaze](https://bitbucket.org/blaze-lib/blaze/overview) v3.7
```

Comment:
Below is a `spack install blaze@3.2`, would you mind updating that as well?

Reply:
Done and squashed in